### ISSUE TITLE
Can you share more info regarding this project?

### ISSUE THREAD

**Issue by france193** [janfiedler/Crypto-Liquidity-Making-Bot#1]
Are you working on it?

**Comment by janfiedler** [comment]
Hi, I have it on my todo list :) Very slow progress.
What more info is interesting for you?This will automate my manual strategy. If price is 0,5 EUR, I put buy order at 0,49, if trade is made, than this amount is set as sell trade for 0,5 so profit is 0,01E per Unit.  It will buy at any level, and sell higer. 

You can control money managment, how much orders at diff levels can be open regarding to price droping. For example open only xy units for price 0,49 and save rest for have possibilty buy ts 0,48; 0,47 ... 0,30

This strategy is for those who believe in IOTA. You can earn money for sell iota higer, or cost averaging price for get more IOTA for lower price. It is win win. 

Secondary effect is you provide liquidity to market and stabilizing price.

**Comment by france193** [comment]
Yes, I understsand this strategy and I like very much! But If the prices rises quickly this is not possible, true? You need a manual control in any case, right? This can be applied only if a currency stay on a certain range for a lot of time.. For example IOTA is between 0,40€ and 0,60€ for a lot of months and this can fit with your strategy! Did you considered also the cost for every exchange? or you refers only to IOTA/€?

On what exchange site are you planning to develop this and using which library? If it is possible I'd like to contribute to this project! 

I don't get your point here : "You can control money managment, how much orders at diff levels can be open regarding to price droping. For example open only xy units for price 0,49 and save rest for have possibilty buy ts 0,48; 0,47 ... 0,30"

Yes, I think that this can also stabilize the price but a big amount sold or bought can exit from your strategy...

**Comment by janfiedler** [comment]
Yes you are absoluty right, this strategy is good for price when is moving in small range. I use this strategy on FOREX for years. I think in future crypto will have same behavior.
You can use it for any other crypto/crypto crypto/fiat pair.

Primary future use for this bot is exchange iota/fiat because we have solution for vending machines (food, drinks, ...). And owners of vending machines want fiat, not iota. So need solution to exchange it for tham for good price.

And when price move to much? You have still your fiat or crypto. Bot never make loose trade. When position is bought, bot save in db time and price. And than will try sell your position only for higer price. If this time never come, he will wait until come. 

For example from manual trade:
![image](https://user-images.githubusercontent.com/13234075/45812964-a0659a00-bcd1-11e8-99fb-419e1124344a.png)

You can see meny buy enter on different price, than price go up, he will sold one by on for bigger price and than again buy for lower price.

First exchange will be coinfalcon, because they are exchange iota/eur and have eur deposit/withdrawals. I was in contact with CEO, it is more trustuble for me than bitfinex, where is to much peoples have problem with fiat transactions.
